fedora_messaging/twisted/protocol.py

```diff
@@ -250,13 +250,13 @@ def _read(self, queue_object, consumer):
                     )
                 else:
                     yield channel.basic_ack(delivery_tag=delivery_frame.delivery_tag)
-                raise e
```

Comment:
Something about this is breaking Python 2.7...? I can't think why off-hand and would be fascinated to learn why.